### PR TITLE
Begrep

### PR DESCRIPTION
**Pull request by kodeebo**
Adds explanations-frontend to editor as iframes.

- Add new concept in article from the toolbar-menu
- Edit existing concept in article
- Search concepts from search page
- Menu option with the whole explanations-frontend iframed

To add or edit concepts locally run a local instance of explanations-frontend, and start editorial with LOCAL_NOTION=true

Also changes auth0 renewAuth to checkSessions and removes silent-callback. Will work when PR in deploy is done

### CODE REVIEW

File: src/components/SlateEditor/plugins/concept/ConceptModal.jsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+      controllable
+      isOpen={true}
+      size="large"
+      backgroundColor="white"
+      minHeight="90vh">
+      {() => (
+        <div css={modalStyles}>
+          <ModalHeader>
+            <StyledHeader>
+              {mode === 'search' && 'Legg til eksisterende begrep'}
```
Comment (chrpeter) [inline src/components/SlateEditor/plugins/concept/ConceptModal.jsx]: Oversettelse?

File: src/components/SlateEditor/plugins/concept/ConceptModal.jsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+              css={iframeStyle}
+            />
+          </ModalBody>
+        </div>
+      )}
+    </Modal>
+  );
+};
+
+const iframeStyle = css`
```
Comment (chrpeter) [inline src/components/SlateEditor/plugins/concept/ConceptModal.jsx]: må ikke dette være defined før det brukes?

File: src/components/SlateEditor/plugins/concept/EditConcept.jsx [hunk 3]
```diff
@@ -1,49 +1,136 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import BEMHelper from 'react-bem-helper';
+import Notion from '@ndla/notion';
+
```
Comment (chrpeter) [inline src/components/SlateEditor/plugins/concept/EditConcept.jsx]: Fjerne space?

File: src/components/SlateEditor/plugins/concept/EditConcept.jsx [hunk 4]
```diff
@@ -1,49 +1,136 @@
...
+      handleError(e);
+    }
   }
 
-  /*  eslint-disable */
-  openEditConcept(e) {
-    e.preventDefault();
-  } /* eslint-enable */
+  AddNewConcept({ data, origin }) {
+    if (origin !== config.explanationFrontendDomain || !data) return;
```
Comment (chrpeter) [inline src/components/SlateEditor/plugins/concept/EditConcept.jsx]: brackets?

File: src/components/SlateEditor/plugins/concept/EditConcept.jsx [hunk 5]
```diff
@@ -1,49 +1,136 @@
...
 
-  async getConcept() {
+  async getConcept(conceptId) {
     const { node, locale } = this.props;
-    const concept = await fetchConcept(node.data.get('content-id'), locale);
-    this.setState({ concept, linkText: node.data.get('link-text') });
+    try {
+      const { data: concept } = await fetchConcept(conceptId, locale);
+      this.setState({ concept, linkText: node.data.get('link-text') });
+    } catch (e) {
```
Comment (chrpeter) [inline src/components/SlateEditor/plugins/concept/EditConcept.jsx]: Kall denne err? Lettere å se med en gang hva det er.

File: src/components/SlateEditor/plugins/concept/EditConcept.jsx [hunk 6]
```diff
@@ -1,49 +1,136 @@
...
+    const {
+      node: { key, text },
+      editor,
+    } = this.props;
+    if (conceptId) {
+      editor.setNodeByKey(key, setConceptDataAttributes({ conceptId, text }));
+    }
+  }
+
+  toggleConceptModal(e) {
```
Comment (chrpeter) [inline src/components/SlateEditor/plugins/concept/EditConcept.jsx]: Kall denne evt? Lettere å se med en gang hva det er.

File: src/config.js [hunk 7]
```diff
@@ -84,6 +84,11 @@ const getAuth0Hostname = () => {
   }
 };
 
+const explanationFrontendDomain = () => {
+  if (process.env.LOCAL_NOTION) return 'http://localhost:3100';
```
Comment (chrpeter) [inline src/config.js]: curly brackets

File: src/config.js [hunk 8]
```diff
@@ -84,6 +84,11 @@ const getAuth0Hostname = () => {
...
+  return `https://explanations-frontend.${ndlaEnvironment}.api.ndla.no`;
```
Comment (chrpeter) [inline src/config.js]: Hva hvis det er prod? Er url'en https://explanations-frontend.prod.api.ndla.no?

File: src/components/SlateEditor/plugins/concept/ConceptModal.jsx (hunk 1 above)
Comment (gunnarvelle) [reply src/components/SlateEditor/plugins/concept/ConceptModal.jsx]: +1

File: src/config.js (hunk 8 above)
Comment (gunnarvelle) [reply src/config.js]: Ja, den er det. I alle fall inntil videre.

File: src/components/SlateEditor/plugins/concept/ConceptModal.jsx (hunk 1 above)
Comment (kodeebo) [reply src/components/SlateEditor/plugins/concept/ConceptModal.jsx]: +2

File: src/components/SlateEditor/plugins/concept/ConceptModal.jsx (hunk 2 above)
Comment (kodeebo) [reply src/components/SlateEditor/plugins/concept/ConceptModal.jsx]: Hoisting :)

**Pull request merged**